test-packages/test-utils/src/utils/download-ui5-resources.ts

```diff
@@ -0,0 +1,115 @@
+// This file can be used to download api.json files for UI5 version we want to add to the test directory.
+// It is run from the download-resources script in the package.json.
+// To get a different version change the version number in the last line of this file.
+import { TestModelVersion } from "../../api";
+import { reduce, forEach, keys } from "lodash";
+import { dirname, resolve } from "path";
+import { writeFileSync, mkdirsSync, readFileSync } from "fs-extra";
```

Comment:
do we need **sync** variants here if all functions in this file are async?

Reply:
Changed